convene-web/app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= page_title %></title>
+    <title><%= page_title %> [Convene Meeting]</title>
```

Comment:
Can we remove "Meeting" from the title? This change is in the global application template, so it will apply to situations where a user might not be in a meeting (e.g. maybe they are administrating their workspace).